vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -78,69 +132,176 @@
                     console.warn("The locale is not supported, using default locale setting(en-US).");
                 }
 
-                let currentDate = false;
+                /** @type {DateHash} */
+                let currentDate;
```

Comment:
This had an unnecessary initial `false` value (undefined works just as well in this case). This was simplified to avoid needing a type like `@type {DateHash|boolean}`.